index.js

```diff
@@ -98,7 +99,7 @@ function send(ctx, path, opts) {
     ctx.set('Last-Modified', stats.mtime.toUTCString());
     ctx.set('Content-Length', stats.size);
     ctx.set('Cache-Control', 'max-age=' + (maxage / 1000 | 0));
-    ctx.type = type(path);
+    ctx.type = typeOverride ? typeOverride : type(path);
     ctx.body = fs.createReadStream(path);
```

Comment:
How? Can you give a proper example? Because as far as I see, `ctx.type` is always overwritten by koa-send.